### DISCUSSION TITLE
Named Sessions?

### DISCUSSION THREAD

**Discussion by josephcrawfordSRH** [rmagatti/auto-session#274] [Q&A] [answered]
Is there a way to have named sessions?  I would prefer to see a project name in telescope vs the full path to every project.  I have yet to find a way but that is the only limitation I see here because many of my projects use some of the same words so searching is a bit rough.

**Comment by rmagatti** [comment]
Custom names for sessions isn't supported currently. The best you can do right now is shorten the path so you don't see the full path. That should work with the sesion-lens options

**Comment by josephcrawfordSRH** [comment]
@rmagatti I am not seeing a good example in the README, I do have session-lens setup and it works with Telescope but I am seeing the full path names vs solely the project name.  I have tried adding the path to `auto_session_suppress_dirs` thinking maybe that is what cuts off the majority of the path but it does not seem to do anything.

![Screenshot 2024-03-07 at 1 18 30 PM](https://github.com/rmagatti/auto-session/assets/100154037/6dfc3bff-46cb-4f4f-8bc5-563a5b5fb805)

**Reply by rmagatti** [reply, marked as answer]
Check this out for a reference on how to use the path_display. PS: if you have rmagatti/session-lens installed, you might want to remove that since auto-session now comes with session-lens built-in.

https://github.com/rmagatti/auto-session/issues/267#issuecomment-1937903688

**Reply by josephcrawfordSRH** [reply]
ahh thank you I have it working, this will work for now.
